Tests/FunctionalTests/ResourcesTests.swift

```diff
@@ -45,4 +45,48 @@ class ResourcesTests: XCTestCase {
                 """)
         }
     }
+
+    func testMovedBinaryResources() {
+        fixture(name: "Resources/Moved") { prefix in
+            var executables = ["SwiftyResource"]
+
+            // Objective-C module requires macOS
+            #if os(macOS)
+            executables.append("SeaResource")
+            #endif
+
+            // Both executables are built in one go since
+            // adding "--target execName" will not produce a binary
```

Comment:
Does that also work when there are no products defined in the package?
Either way, I think building both products shouldn't hurt. Especially since now the binaries and resources are moved...

Reply:
> Does that also work when there are no products defined in the package?
> Either way, I think building both products shouldn't hurt. Especially since now the binaries and resources are moved...

SwiftPM actually infers a executable product for each executable target, so technically there is a product anyway.  But I don't think there is anything to change here, since building both is fine.

Reply:
Since this failed on Linux trying to build the ObjC target as well, I'm now building each product separately (and with that the ObjC target is not built on Linux).